acceptance/framework/consul/consul_cluster.go

```diff
@@ -47,6 +47,8 @@ type HelmCluster struct {
 	noCleanupOnFailure bool
 	debugDirectory     string
 	logger             terratestLogger.TestLogger
+
+	ACLToken string
```

Comment:
Can we move this public field to the top of the struct?

Reply:
Oh, also a nit, but what do you think of `ACLTokenOverride` to communicate that there is behavior whether this is set or not. Maybe add a comment that this is optional. Right now, someone using this would have to read the implementation in order to know that the cluster has behavior with and without this value being set.

Reply:
I think I prefer to keep it as `ACLToken` because it's also not an override. It's more of an optional value that you can set if you don't want us to get the token from the kube secret. 

I've added docs to clarify what it should be used for.

Reply:
Cool, I dig that. I'll use that pattern in my work in the codebase in the future too.